rollup-bundle.js

```diff
@@ -0,0 +1,35 @@
+const rollup = require('rollup');
+const nodeResolve = require('rollup-plugin-node-resolve');
+const commonjs = require('rollup-plugin-commonjs');
+
+const base = `${__dirname}/bazel-out/darwin-fastbuild/bin/src/bazel-out/darwin-fastbuild/bin/src`;
+
+class NormalizePaths {
+  // TODO Fix import statements in ngfactories to avoid this path plugin
+  resolveId(importee, importer) {
```

Comment:
if we didn't have to patch filepaths, is there a way to just call the rollup CLI without having our own main like this?